demos/collection/grid.php

```diff
@@ -1,13 +1,24 @@
 <?php
 
-require_once __DIR__ . '/init.php';
-require_once __DIR__ . '/database.php';
+chdir('..');
```

Comment:
Is `chdir` here only for the include below? If yes, use `require_once __DIR__ . '/..' ...`

Reply:
if chdir is not use than atk-init.php is not able to include database.php and somedatadef.php
not sure what is the best way to handle this.

I can also still use chdir and do:
````
chdir('..');
require_once __DIR__ . '/../atk-init.php';
````
what do you think is best?

Reply:
Never use `chdir`, we are not in bash ;-), and use `__DIR__` everywhere - also in `atk-init.php` which should work then.

Reply:
fix